docs/cookbook/examples/ami_version_packager.rst

```diff
@@ -253,5 +253,5 @@ It is intended to be used in conjunction with the script dicussed in :ref:`ami_h
             raise ShotgunException("Unknown action... :%s" % sa.action)
             
         
-        print "\nVersion Packager done!"
+        print("\nVersion Packager done!")
```

Comment:
sphinx docs still say 

```
********************
Minimum Requirements
********************

- Shotgun server v5.4.14 or higher for API v3.0.16+.
- Shotgun server v2.4.12 or higher for API v3.0.8.
- Shotgun server v1.8 or higher for API v3.0.7.
- Python v2.4 - v2.7. (We do not currently support Python 3)

```

Please do a quick pass to eyeball that they sphinx docs are up to date!

Reply:
anything we should add to the sphinx docs specifically around py3?

Reply:
awesome!